pkg/lintcontext/create_contexts.go

```diff
@@ -50,6 +51,12 @@ func CreateContextsWithOptions(options Options, filesOrDirs ...string) ([]LintCo
 			continue
 		}
 
+        for _, path := range ignorePaths {
+            if strings.HasPrefix(fileOrDir, path) {
+                continue fileOrDirsLoop
+            }
+        }
```

Comment:
This is a naive solution to do prefix check by iterating `ignorePaths` slice. Let me know if you have suggestions for more optimized solution 🤔